client/src/components/Nav/Nav.jsx

```diff
@@ -1,11 +1,80 @@
 import React, { Component } from 'react';
+import './Nav.css';
+
+var styles = {
+	display: 'none',
+	marginTop: 120
+};
 
 class Nav extends Component {
-   render() {
-      return (
-         <div>Navbar here</div>
-      )
-   }
+	constructor(props){
+		super(props);
+		this.toggleDropdown = this.toggleDropdown.bind(this);
+		this.state = {
+			show: false
+		};
+	}
+
+	// toggle mobile menu on ham click
+	toggleDropdown() {
+		// test
+		console.log('hello!');
+		
+		if(this.state.show === false){
+			styles = {
```

Comment:
Don't set styles here. Keep styles in a CSS class and in `render()`, conditionally set the className to whichever class you want it to have depending on the state.